python/widget-store/README.md

```diff
@@ -34,7 +34,7 @@ Then run:
 
 ```shell
 pip install -r requirements.txt
-dbos migrate
+alembic upgrade head
```

Comment:
following the instructions above (which say to set PGPASSWORD env var) do not appear to work

Reply:
We need to include instructions or a script to create the database before calling alembic